src/tests/Common/helixpublishwitharcade.proj

```diff
@@ -197,9 +197,20 @@
 
       <ReducedLegacyPayloadFilesFinal Include="@(ReducedLegacyPayloadFiles)" Condition=" '%(Extension)' != '.sh' and '$(TestWrapperTargetsWindows)' == 'true' "/>
       <ReducedLegacyPayloadFilesFinal Include="@(ReducedLegacyPayloadFiles)" Condition=" '%(Extension)' != '.cmd' and '$(TestWrapperTargetsWindows)' != 'true' "/>
+      <!-- Don't include the new merged wrappers in the legacy builds -->
       <ReducedLegacyPayloadFilesFinal Remove="@(_MergedWrapperRunScript)" />
     </ItemGroup>
 
+    <PropertyGroup Condition="'$(TargetsMobile)' == 'true' or '$(TargetsBrowser)' == 'true'">
+      <!-- When targeting mobile or browser runs, we publish self-contained builds and build mobile/wasm apps, so we want to exclude these files from the legacy payloads as well -->
```

Comment:
I am kind of confused why we want to exclude these files? The ones in publish I understand, but why the AppBundle?

Reply:
We run the merged payloads in a completely separate mechanism from how the legacy payloads run, so we don't want to accidentally include the merged payloads (including the apps that we're actually going to run on the device) in the legacy payloads. The AppBundle folders would only serve to bloat the zip we send over to Helix, which we don't want to do.